packages/fxa-auth-server/test/local/routes/devices-and-sessions.js

```diff
@@ -973,7 +973,6 @@ describe('/account/device/destroy', () => {
       return runTest(route, mockRequest, () => {
         assert.equal(mockDB.deleteDevice.callCount, 1);
         assert.isFalse(mockLog.error.calledOnceWith('deviceDestroy.revokeRefreshToken.error'));
-        assert.isTrue(mockOAuthDb.revokeRefreshTokenById.calledOnceWith(refreshTokenId));
```

Comment:
Sorry I didn't notice this earlier, but we should be back these assertions. You can mock the response by the `db.deleteDevice` method by adding this before each test runs.

```
mockDb.deleteDevice = sinon.spy(async () => refreshTokenId);
```

Reply:
Thanks @vbudhram  
I updated the test.